FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +43,53 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] httpClientResult in
+
+			guard self != nil else { return }
+
+			var result: FeedLoader.Result
+
+			let feedLoaderResult = Self.httpClientResult2RemoteFeedLoaderResult(httpClientResult)
+			switch feedLoaderResult {
+			case .success(let responseData):
+				result = Self.responseData2FeedLoaderResult(responseData)
+
+			case .failure(let error):
+				result = .failure(error)
+			}
+
+			completion(result)
+		}
+	}
+
+	private static func httpClientResult2RemoteFeedLoaderResult(_ httpClientResult: HTTPClient.Result) -> RemoteFeedLoaderResult {
```

Comment:
Please rename all methods to better follow the swift [guidelines](https://swift.org/documentation/api-design-guidelines/):

(I will suggest names for these functions, and will refer to them by their suggested names on follow-up comments)
```suggestion
	private static func remoteFeedLoaderResult(from httpClientResult: HTTPClient.Result) -> RemoteFeedLoaderResult {
```